cluster-scope/overlays/prod/moc/infra/multiclusterobservability/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - observability.yaml
```

Comment:
I don't think we expect multiple resources of the same kind in this overlay... Either please move it to the `cluster-scope/base` or remove the `infra/multiclusterobservability/kustomization.yaml` and include it directly as `infra/multiclusterobservability/observability.yaml` in `infra/kustomization.yaml`